Python/ceval.c

```diff
@@ -4859,12 +4859,12 @@ import_all_from(PyObject *locals, PyObject *v)
             }
             if (!PyUnicode_Check(modname)) {
                 PyErr_Format(PyExc_TypeError,
-                             "__name__ must be a string, not %.100s",
+                             "module.__name__ must be a string, not %.100s",
```

Comment:
why? I don't get the point.

Reply:
Because "module" is not a name of any object.

But it was just a question. I don't know what is better in English. If `.` looks better to you, keep it.

Reply:
I think either is fine in practice, but leaving `.` out would technically be more accurate (since we're using "module" as an ordinary English word, rather than as a variable name)

Reply:
Thanks!